pmm-app/tests/pages/pmmSettingsPage.js

```diff
@@ -286,4 +297,31 @@ module.exports = {
     }
     I.see(ruleName);
   },
+
+  async verifyTooltip(tooltipObj) {
+    I.see(tooltipObj.text, this.fields.tooltipLocator);
+    I.seeAttributesOnElements(`${this.fields.tooltipLocator} > div > a`, {href: tooltipObj.link});
```

Comment:
This locator can be improved, no other way to handle it?

Reply:
do you think we need to do such changes? the method is pretty simple and reusable now